internal/object/object.go

```diff
@@ -199,3 +201,13 @@ func NewIngress(name, ns, backendName string, backendPort int) *networkingv1.Ing
 		},
 	}
 }
+
+// TagFromImage returns the tag from the provided image or an
+// empty string if the image does not contain a tag.
+func TagFromImage(image string) string {
+	if strings.Contains(image, ":") {
+		parsed := strings.Split(image, ":")
+		return parsed[1]
+	}
+	return ""
```

Comment:
Well should this return `image` instead of ""?

Reply:
@stevesloka I don't think so. The purpose of the func is to return the tag from the provided image. If the image does not contain a tag, I would expect an empty tag, i.e. "" or an error.